src/Proto.Actor/Context/ActorContext.cs

```diff
@@ -341,13 +341,11 @@ private ValueTask InternalInvokeUserMessageAsync(object msg)
                 System.DeadLetter.SendUserMessage(Self, msg);
                 return default;
             }
+            var influenceReceiveTimeout = MessageEnvelope.UnwrapMessage(msg) is not INotInfluenceReceiveTimeout;
```

Comment:
this introduces an unwrap in a hot-path.
What are the effects on in process benchmark?

Reply:
That is a good point, will do a minor change to fix it